src/Shell/Helper/ProgressHelper.php

```diff
@@ -35,6 +35,17 @@
  */
 class ProgressHelper extends Helper
 {
+    /**
+     * Default value for progress bar total value.
+     * Percent completion is derived from progress/total
+     */
+    private const DEFAULT_TOTAL = 100;
```

Comment:
shouldnt those be protected to be consistent with the rest?

Reply:
That's fair, I'll update the new constants.